.env.example

```diff
@@ -20,9 +20,10 @@ POSTGRES_URL="postgresql://${POSTGRES_USER}:${POSTGRES_PASSWORD}@localhost:5432/
 
 ## MongoDB
 
-MONGO_USER=
-MONGO_PASSWORD=
-MONGO_URL="mongodb://${MONGO_USER}:${MONGO_PASSWORD}@localhost:27017/db?authSource=admin"
+# MONGO_USER=
```

Comment:
`env.mjs` is throwing the following, maybe should include the mongo user and password in the example file?
![image](https://github.com/CS3219-AY2324S1/ay2324s1-course-assessment-g39/assets/70463178/d589e794-1ee4-4219-aac9-c1e5f1747810)
Theres a comma coz theres one more var I forgot to include